example/test/exampleFluentDslWithTypeScript.spec.ts

```diff
@@ -1,7 +1,17 @@
 import {browser, element, by} from 'protractor';
-import {useCase, scenario, step} from '../../lib';
-import {ScreenAnnotationStyle} from "../../lib";
-import {ScreenAnnotationClickAction} from "../../lib";
+// when using the npm module, use `import {...} from 'scenarioo'` instead
+import {
+  useCase,
+  scenario,
+  step,
+  fluentDslConfig,
+  ScreenAnnotationStyle,
+  ScreenAnnotationClickAction,
+  ScreenAnnotationRegion
+} from '../../lib';
+
+// Note that labels have to be registered before usage. Usually we do this once on starting up the test suite.
```

Comment:
will make this docu even more prominent and even in the header of the file, explaining whole example.
Also same for non-typsescript Fluent DSL example.